package.json

```diff
@@ -23,15 +23,18 @@
     "test:int": "npm run compile && jasmine build/spec/journal.spec.js"
   },
   "dependencies": {
-    "@types/jwt-decode": "^2.2.1",
     "aws-sdk": "^2.389.0",
-    "moment": "^2.23.0"
+    "moment": "^2.23.0",
+    "@hapi/joi": "^15.0.3"
   },
   "devDependencies": {
     "@dvsa/mes-journal-schema": "^1.0.7",
```

Comment:
Does journal schema need to be 1.1.0 for develop?

Reply:
Good catch !